float64 x
float64 y
---
HDMap/RoadMSG curr_road
HDMap/JunctionMSG junction
HDMap/RoadMSG next_road